src/i18n/strings/en_US.json

```diff
@@ -231,6 +231,7 @@
     "demote": "demote",
     "Deops user with given id": "Deops user with given id",
     "Default": "Default",
+    "Delete widget": "Delete widget",
```

Comment:
Ftr, these will just fall back to the en_EN strings if they're not found so there's no need to fill them in if they're the same.